admin/services/data-access/payload.data.service.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const R = require('ramda')
+const sqlService = require('./sql.service')
+
+const payloadDataService = {
+  /**
+   * Retrieve a check payload.  Case insensitive.
+   * @param checkCode
+   * @return {Promise<any>}
+   */
+  sqlFindOneByCheckCode: async function sqlFindOneByCheckCode (checkCode) {
+    const sql = `select cr.payload 
+                 from [mtc_admin].[check] chk 
+                 join [mtc_admin].[checkResult] cr on chk.id = cr.check_id
+                 where chk.checkCode = @checkCode`
+    const params = [
+      { name: 'checkCode', value: checkCode, type: sqlService.TYPES.UniqueIdentifier }
+    ]
+    const res = await sqlService.query(sql, params)
+    return JSON.parse(R.prop('payload', R.head(res)))
```

Comment:
if the check can't be found would this throw or just return an empty `{}`?

Reply:
It throws from sqlQuery.  The exception is caught in the controller - try/catch wrapped around the `getPayload` method.  It then returns a check not found JSON object.